### DISCUSSION TITLE
auto fetch stripe plans

### DISCUSSION THREAD

**Discussion by himeshp** [Etheonor/supanextail#25] [Feedback]
to fetch stripe plans automatically.
currently the plans I made from my account, does not reflect in the ui.

**Comment by himeshp** [comment]
is there any plans for supporting currency, I added plans as INR into my stripe account.

**Comment by Etheonor** [comment]
Does that means the pricing page would be SSR?

Right now the user have to add price list IDs in utils/priceList.js. But it may be not the best idea and I'm checking the Pricing component right now, the user need to configure a lot of things so there are definitely areas for improvement.

**Reply by himeshp** [reply]
ssr yes, so plans are auto fetch and user can jus select which to display.
also multi-currency support.
